mass_mailing_list_dynamic/views/mail_mass_mailing_list_view.xml

```diff
@@ -61,6 +70,16 @@
                     />
                 </group>
             </xpath>
+            <field name="contact_nbr" position="attributes">
+                <attribute name="attrs">
+                    {'invisible': [('unsynced', '=', True)]}
+                </attribute>
+            </field>
+            <field name="contact_nbr" position="after">
+                <span attrs="{'invisible': [('unsynced', '=', False)]}">
```

Comment:
I'd appreciate it if you can rename the field to `synced` and flip all the logic. Thinking about "unsynced" and "not unsynced" lists produces more head-scratches that thinking about "synced" and "not synced" IMHO :thinking: